venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_tabs.js

```diff
@@ -95,7 +99,13 @@ function renderConferenceHeader() {
     location: "Vancouver Convention Center, Vancouver, BC, Canada",
     date: "April 30 - May 3, 2018",
     website: "http://www.iclr.cc",
-    instructions: null,  // Add any custom instructions here. Accepts HTML
+    instructions: `<b>Important Information about Anonymity:\n</b>
```

Comment:
@melisabok @zbialecki I'm having trouble getting this to format correctly. any idea what I should be doing differently?

Reply:
I think \n is not html code.

Reply:
Also, for all JS code running in the browser, we shouldn't use ES6 features like template strings. I have been making sure all the code runs on browsers back to IE11, you can refer to caniuse.com if you have questions about a specific API.

One other note, to make text bold use the `<strong>` tag not `<b>`